js/stateless.js/src/idls/index.ts

```diff
@@ -9,7 +9,10 @@ import {
     IDL as LightCompressedPdaIDL,
     LightCompressedPda,
 } from './light_compressed_pda';
-import { IDL as LightUserRegistryIDL, LightUserRegistry } from './light_user_registry';
+import {
+    IDL as LightUserRegistryIDL,
+    LightUserRegistry,
+} from './light_user_registry';
```

Comment:
in line 7 of same file, we need to change 
`import { IDL as LightSystemIDL, LightRegistry } from './light_registry';`
to 
`import { IDL as LightRegistryIdl, LightRegistry } from './light_registry';`
as well.